browser/brave_wallet/brave_wallet_prefs_unittest.cc

```diff
@@ -274,4 +276,79 @@ TEST_F(BraveWalletPrefsUnitTest, MigrateShowTestNetworksToggle) {
   EXPECT_FALSE(GetPrefs()->HasPrefPath(kShowWalletTestNetworksDeprecated));
 }
 
+TEST_F(BraveWalletPrefsUnitTest, MigrateShowChainIdNetworkInfo) {
+  GetPrefs()->SetBoolean(kBraveWalletUserAssetEthContractAddressMigrated, true);
+  GetPrefs()->SetBoolean(kBraveWalletUserAssetsAddPreloadingNetworksMigrated,
+                         true);
+  GetPrefs()->SetBoolean(kBraveWalletUserAssetsAddIsNFTMigrated, true);
+  GetPrefs()->SetBoolean(kBraveWalletEthereumTransactionsCoinTypeMigrated,
+                         true);
+  GetPrefs()->SetBoolean(kBraveWalletDeprecateEthereumTestNetworksMigrated,
+                         true);
+
+  EXPECT_FALSE(
+      GetPrefs()->HasPrefPath(kBraveWalletTransactionsChainIdMigrated));
+  EXPECT_FALSE(GetPrefs()->GetBoolean(kBraveWalletTransactionsChainIdMigrated));
+
+  const char ethTxId[] = "b1e8dda1";
+  const auto ethPath = base::StringPrintf("%s.%s", "mainnet", ethTxId);
+  base::Value::Dict tx1;
+  tx1.Set("id", ethTxId);
+
+  const char solTxId[] = "887e878f";
+  const auto solPath = base::StringPrintf("%s.%s", "devnet", solTxId);
+  base::Value::Dict tx2;
+  tx2.Set("id", solTxId);
+
+  const char filTxId[] = "197ea1e5";
+  const auto filPath = base::StringPrintf("%s.%s", "testnet", filTxId);
+  base::Value::Dict tx3;
+  tx3.Set("id", filTxId);
+
+  {
+    ScopedDictPrefUpdate update(GetPrefs(), kBraveWalletEthereumTransactions);
+    update->SetByDottedPath(ethPath, tx1.Clone());
```

Comment:
Seems like it can be move instead of clone since it's not used later? Same thing for tx2 and tx3.

Reply:
fixed